Libs/MRML/Widgets/Testing/qMRMLNodeComboBoxTest9.cxx

```diff
@@ -0,0 +1,158 @@
+/*==============================================================================
+
+  Program: 3D Slicer
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  This file was originally developed by Johan Andruejol, Kitware Inc.
+
+==============================================================================*/
+
+// QT includes
+#include <QApplication>
+#include <QTimer>
+
+// qMRML includes
+#include "qMRMLNodeComboBox.h"
+
+// MRML includes
+#include <vtkMRMLNode.h>
+#include <vtkMRMLScene.h>
+#include <vtkMRMLScalarVolumeNode.h>
+
+// VTK includes
+#include <vtkNew.h>
+
+// test the filtering with many cases
+int qMRMLNodeComboBoxTest9( int argc, char * argv [] )
+{
+  QApplication app(argc, argv);
+
+  vtkNew<vtkMRMLScene> scene;
+
+  vtkNew<vtkMRMLScalarVolumeNode> noAttributeNode;
+  scene->AddNode(noAttributeNode.GetPointer());
+
+  const char *testingAttributeName = "testingAttribute";
+  const char *testingAttribute = noAttributeNode->GetAttribute(testingAttributeName);
+  std::cout << "Volume node with no call to SetAttribute, GetAttribute returns " << (testingAttribute ? testingAttribute : "0") << "." << std::endl;
+  if (testingAttribute != NULL)
```

Comment:
Please, consider using the macros provided by `ctkCoreTestingMacros.h`

For an example of use, see https://github.com/commontk/CTK/blob/afe8505a8286a701900fca1c97a4ad8876bbbe1a/Libs/Widgets/Testing/Cpp/ctkAxesWidgetTest1.cpp#L91-L107